modular_citadel/code/modules/research/techweb/all_nodes.dm

```diff
@@ -0,0 +1,8 @@
+/datum/techweb_node/computer_board_gaming
```

Comment:
override the existing node that has arcade machines.

Reply:
as in define it again here and put vars to override said vars.

Reply:
Is this not overriding it? That was intentional.
It adds it to the one that lets you make arcade machines. It's for slackers after all.

Reply:
I don't quite understand what the issue is with this one. It works in testing exactly as I wanted it to.